test/colony-network-staking.js

```diff
@@ -258,6 +258,28 @@ contract("ColonyNetworkStaking", accounts => {
       const rootHashNNodes = await colonyNetwork.getReputationRootHashNNodes.call();
       assert(rootHashNNodes.equals(10));
     });
+
+    it("should not allow someone who is not ColonyNetwork to appendReputationUpdateLog", async () => {
```

Comment:
Maybe tests should move to `reputation-update.js` for functionality moved from `ColonyNetwork` to `ReputationMiningCycle`

Reply:
I've ended up renaming `colony-network-staking.js` to `colony-network-mining.js`, which I think more accurately captures what the tests in that file have become, as well as the fact that they now span `ReputationMiningCycle` and `ColonyNetwork`.